modules/cco-ccoctl-upgrading.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-upgrading_{context}"]
+= Updating AWS resources with the Cloud Credential Operator utility
+
+The process for upgrading an {product-title} cluster configured for manual mode with STS is similar to installing on a cluster for which you create the AWS resources individually.
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. To place JSON files on the local file system instead, use the `--dry-run` flag. These JSON files can be reviewed or modified and then applied with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* Obtain the {product-title} release image for the version that you are upgrading to.
+
+* Extract and prepare the `ccoctl` binary from the release image.
+
+.Procedure
+
+. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc adm release extract --credentials-requests --cloud=aws --to=__<path_to_directory_with_list_of_credentials_requests>__/credrequests quay.io/__<path_to>__/ocp-release:__<version>__
+----
+
+. For each `CredentialsRequest` CR in the release image, ensure that a namespace exists in the `spec.secretRef.namespace` field. This field is where the generated secrets that hold the credentials configuration are stored.
```

Comment:
Maybe way too pedantic, but this is all subtle enough as it is and I prefer to err on the side of over-explanation :)
```suggestion
. For each `CredentialsRequest` CR in the release image, ensure that a namespace exists in the cluster matching the text in the `spec.secretRef.namespace` field. This field is where the generated secrets that hold the credentials configuration are stored.
```